config/datacite.yml

```diff
@@ -0,0 +1,18 @@
+---
+development:
+  domains:   '*'
+  login:    'DATACITE.DCE'
+  password: 'mahoniaD0I'
+  prefix:   '10.24354'
+
+test:
+  domains:   '*'
+  login:    'DATACITE.DCE'
+  password: 'mahoniaD0I'
+  prefix:   '10.5072'
+
+production:
+  domains:   '*'
+  login:    'DATACITE.DCE'
+  password: 'mahoniaD0I'
```

Comment:
Can we use dotenv to manage these? We shouldn't check passwords into github, right?

Reply:
:+1: